.github/workflows/ci.yml

```diff
@@ -11,6 +11,8 @@ jobs:
         uses: actions/setup-python@v2
         with:
           python-version: "3.5"
+        env:
+          PIP_TRUSTED_HOST: "pypi.python.org pypi.org files.pythonhosted.org"
 
```

Comment:
This change fixes Python 3.5 setup fails with an invalid certificate when running pip. It's reported in `actions/setup-python`: https://github.com/actions/setup-python/issues/866